firmwares/zooz/ZSE42.json

```diff
@@ -0,0 +1,23 @@
+{
+	"devices": [
+		{
+			"brand": "Zooz",
+			"model": "ZSE42",
+			"manufacturerId": "0x027a",
+			"productType": "0x7000",
+			"productId": "0xe002",
+			"firmwareVersion": {
+				"min": "1.0",
+				"max": "1.255"
+			}
+		}
+	],
+	"upgrades": [
+		{
+			"version": "1.50",
```

Comment:
version format: x.y.z
for Zooz product details contact @[TSHAdmin]

Reply:
Happy to change the format, but here we have version 1.50 https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/

Is the zooz firmware page using the wrong firmware format?

Reply:
For [ZSE42_V01R50_US.gb](https://www.getzooz.com/firmware/ZSE42_V01R50_US.gbl)l firmware version is 1.50.1
If you like to get more details on versioning of Zooz products contact @[TSHAdmin]

Reply:
Thank you, added the patch version.